scene/main/node.h

```diff
@@ -504,6 +510,9 @@ class Node : public Object {
 	Node *get_parent() const;
 	Node *find_parent(const String &p_pattern) const;
 
+	void set_unique_scene_id(int32_t p_unique_id);
+	int32_t get_unique_scene_id() const;
```

Comment:
Since these are trivial, they could be implemented inline.

Reply:
There is not a lot inline in Node to keep the API clean to read, with the exception of code that really needing the inline to be faster.